src/Storages/TTLDescription.h

```diff
@@ -0,0 +1,97 @@
+#pragma once
+#include <Parsers/IAST_fwd.h>
+#include <Storages/DataDestinationType.h>
+#include <Storages/ColumnsDescription.h>
+#include <Interpreters/ExpressionActions.h>
+#include <Interpreters/AggregateDescription.h>
+#include <Storages/StorageInMemoryMetadata.h>
+#include <Storages/TTLMode.h>
+
+namespace DB
+{
+
+struct TTLSetPartDescription
```

Comment:
I think this name is a bit confusing. Looks like it's related to data parts. Maybe name it somehing like `TTLAggregateDescription`

Reply:
Ok, it was my first idea)